deploy/conf/staging-gs/site.conf

```diff
@@ -44,23 +44,8 @@ observe-engine {
   #   full: connect and command the system
   #   readOnly: connect, but only to read values
   #   simulated: don't connect, simulate internally
-  # You can override in local secrets.conf
   system-control {
-      dhs = full
-      flamingos-2 = simulated
-      gcal = full
-      ghost = simulated
-      ghost-gds = simulated
-      gmos = full
-      gnirs = simulated
-      gpi = simulated
-      gpi-gds = simulated
-      gsaoi = simulated
-      gws = full 
-      nifs = simulated
-      niri = simulated
-      tcs = full
-      gems = simulated
+      # Systems should be configured in local systems.conf file.
```

Comment:
is the fallback all simulated?

Reply:
yes, it's the one in `base.conf`; the same one that's used in dev